<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="outputbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>551</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="inputbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>129</y>
      <width>551</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="msg2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>451</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="msg1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>451</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendmsg2_button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>send msg 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendmsg1_button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>310</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>send msg 1</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
